packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -0,0 +1,387 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Script } from "forge-std/Script.sol";
+import { console2 as console } from "forge-std/console2.sol";
+
+import { Artifacts } from "scripts/Artifacts.s.sol";
+import { DeployConfig } from "scripts/DeployConfig.s.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { L1StandardBridge } from "src/L1/L1StandardBridge.sol";
+import { L1CrossDomainMessenger } from "src/L1/L1CrossDomainMessenger.sol";
+import { L2StandardBridge } from "src/L2/L2StandardBridge.sol";
+import { L2CrossDomainMessenger } from "src/L2/L2CrossDomainMessenger.sol";
+import { SequencerFeeVault } from "src/L2/SequencerFeeVault.sol";
+import { FeeVault } from "src/universal/FeeVault.sol";
+import { OptimismMintableERC20Factory } from "src/universal/OptimismMintableERC20Factory.sol";
+import { L1Block } from "src/L2/L1Block.sol";
+import { GovernanceToken } from "src/governance/GovernanceToken.sol";
+import { EIP1967Helper } from "test/mocks/EIP1967Helper.sol";
+
+interface IInitializable {
+    function initialize(address _addr) external;
+}
+
+/// @dev The general flow of adding a predeploy is:
+///      1. _setProxies uses vm.etch to set the Proxy.sol deployed bytecode for proxy address 0x420...0 to 0x420...799.
```

Comment:
I prefer the first suggestion, but am ok with either
```suggestion
///      1. _setProxies uses vm.etch to set the Proxy.sol deployed bytecode for proxy address `0x420...000` to `0x420...000 + PROXY_COUNT`.
```
```suggestion
///      1. _setProxies uses vm.etch to set the Proxy.sol deployed bytecode for proxy address 0x420...0 to 0x420...7ff.
```

Reply:
Committed first suggestion